k8s/README.md

```diff
@@ -31,10 +31,34 @@ kubectl apply -f openebs-config.yaml
 ### (Optional) Enable monitoring using prometheus and grafana
 
 Use this step if you don't already have monitoring services installed
-on your k8s cluster. 
+on your k8s cluster.
 
 ```
 kubectl apply -f openebs-monitoring-pg.yaml
 ```
 
-This folder also contains a sample openebs volume monitoring dashboard (openebs-pg-dashboard.json) that can be imported into your grafana. 
+This folder also contains a sample openebs volume monitoring dashboard (openebs-pg-dashboard.json) that can be imported into your grafana.
+
+### (Optional) Enable monitoring using Prometheus Operator
+
+It is assumed that you have prometheus operator already deployed and working fine. If not done already follow instructions to do it [here](https://github.com/helm/charts/tree/master/stable/prometheus-operator#installing-the-chart).
+
+While deploying above chart make sure the values of specified field are as follows:
+
+```yaml
+    serviceMonitorSelectorNilUsesHelmValues: false
+    serviceMonitorSelector: {}
+    serviceMonitorNamespaceSelector: {}
+```
+
+This makes sure that all the [`ServiceMonitor` objects](https://github.com/coreos/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md#related-resources) from all namespaces we create further will be selected.
+
+Now to monitor openebs related resources create following `ServiceMonitor` object in `openebs` namespace. See [openebs-servicemonitor.yaml](openebs-servicemonitor.yaml) file.
+
+```bash
+kubectl -n openebs apply -f openebs-servicemonitor.yaml
+```
+
+Find [docs here](https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#servicemonitor) to read about other fields in `ServiceMonitor` object.
+
+Now once this config is picked up by prometheus operator it will start scraping the metrics and you can start seeing them in prometheus dashboard. The metrics relevant to openebs are conveniently prefixed with `openebs*`.
```

Comment:
Are there any other metrics which don't start with `openebs*` ?

Reply:
yes, we have also instrumented couple of components of openebs other than pvc's such m-apiserver, where metrics start with prefix `latest_openebs*` but we may change it to `openebs*`.
cc: @kmova @AmitKumarDas

Reply:
@utkarshmani1997 can you raise a new issue to include `openebs*` as the metrics prefix in m-apiserver. IMO lets keep old prefix and at the same time add this new one.

Reply:
for now also included `latest_openebs*`, probably remove them when that metric is changed to be just `openebs*`